dashboard/app/helpers/script_levels_helper.rb

```diff
@@ -36,9 +38,21 @@ def next_progression_level_or_redirect_path(script_level)
         script_level.next_progression_level
       end
 
-    next_level ?
-        build_script_level_path(next_level) :
-        script_completion_redirect(script_level.script)
+    if script_level.level.try(:is_plc_evaluation?)
+      last_result = UserLevel.where(user: current_user, level: script_level.levels[0]).last
+      level_source = last_result ? LevelSource.where(id: last_result.level_source_id).last : nil
+      enrolled_modules = determine_preferred_learning_modules level_source, script_level.script
+      enrollment_unit_assignment = Plc::EnrollmentUnitAssignment.find_by(user: current_user, plc_course_unit: script_level.script.plc_course_unit)
+      if enrollment_unit_assignment
+        preview_assignments_path(unit_assignment_id: enrollment_unit_assignment, params: {enrolled_modules: enrolled_modules})
```

Comment:
I thought we discussed in person that these shouldn't be sent via URL params?  Instead the logic for figuring out which modules to suggest should happen on the override page.

Right now we're doing all the calculation to figure out which modules to suggest inside the redirect.  Just redirect and calculate which modules were suggested on the override page.